src/main/java/lux/fm/bookingservice/model/User.java

```diff
@@ -57,7 +57,7 @@ public class User implements UserDetails {
     private Set<Role> roles = new HashSet<>();
 
     @Column(name = "is_deleted", nullable = false)
-    private boolean isDeleted;
+    private boolean isDeleted = false;
```

Comment:
Do we need this? User is_deleted column already has defaultBooleanValue false in changeset 1